src/tiled/objecttypesmodel.cpp

```diff
@@ -106,6 +108,21 @@ Qt::ItemFlags ObjectTypesModel::flags(const QModelIndex &index) const
     return f;
 }
 
+static bool objectTypeLessThan(ObjectType const &a, ObjectType const &b)
```

Comment:
Please change this to `const ObjectType &` to keep with the same style used in the rest of the code.